roles/php/tasks/configure-apache2.yml

```diff
@@ -16,18 +16,3 @@
 - name: php | Run the tasks for the Apache PHP module
   include_tasks: mod-php.yml
   when: modphp.stat.exists
-
-- name: php | disable php apache module
-  apache2_module:
-    state: absent
-    name: php{{ php_eolversion }}
-  notify: restart apache
-  when:
-    - running_on_server
-  ignore_errors: true
-
-- name: php | enable php apache module
-  apache2_module:
-    state: present
-    name: php{{ php_version }}
-  notify: restart {{ php_webserver }}
```

Comment:
Why do we remove these tasks?

Reply:
ansible does the right thing and installs the module.